test/topology_custom/test_multidc.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test functionality on the cluster with different values of the --smp parameter on the nodes.
+"""
+import logging
+import time
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables
+from test.pylib.util import unique_name
+from test.topology.util import wait_for_token_ring_and_group0_consistency
+import pytest
+
+logger = logging.getLogger(__name__)
+
+# Checks a cluster boot/operations in multi-dc environment
+@pytest.mark.asyncio
+async def test_multidc(manager: ManagerClient, random_tables: RandomTables) -> None:
```

Comment:
```
-async def test_multidc(manager: ManagerClient, random_tables: RandomTables) -> None:
+async def test_multidc(request: FixtureRequest, manager: ManagerClient) -> None:
```

Reply:
The `random_tables` fixture needs a live cluster to initialize. This is the cause of the error.

Reply:
Yep.